src/docs/development/platform-integration/platform-channels.md

```diff
@@ -119,10 +119,10 @@ Start by creating a new app:
 
 * In a terminal run: `flutter create batterylevel`
 
-By default our template supports writing Android code using Java, or iOS code
-using Objective-C. To use Kotlin or Swift, use the `-i` and/or `-a` flags:
+By default our template supports writing Android code using Kotlin, or iOS code
```

Comment:
It looks like this change removes Java examples. I know Kotlin is the default, but should we really remove the Java examples? We provided side-by-side Java/Kotlin examples elsewhere. Is there a reason why this guide should be different?

Reply:
I didn't remove java. It's under a tab.